src/neuroconv/datainterfaces/ecephys/ced/ceddatainterface.py

```diff
@@ -9,12 +9,12 @@ def _test_sonpy_installation() -> None:
     get_package(package_name="sonpy", excluded_platforms_and_python_versions=dict(darwin=["3.7"]))
 
 
-class CEDRecordingInterface(BaseRecordingExtractorInterface):
+class CEDRecordingInterface(
+    BaseRecordingExtractorInterface,
+):
     """Primary data interface class for converting data from CED (Cambridge Electronic Design)."""
 
-    _test_sonpy_installation()
-    spikeinterface = get_package(package_name="spikeinterface")
-    RX = spikeinterface.extractors.CedRecordingExtractor
+    RXName = "CedRecordingExtractor"
```

Comment:
Most of the interfaces could have extractor names auto-inferred, but this and Neuroscope (SI: 'NeuroScope') are some of the conditions where a manual name override is necessary and not difficult to do.